scenarios/reproducers/3-nodes.yml

```diff
@@ -9,11 +9,11 @@ cifmw_path: "{{ ansible_user_dir }}/.crc/bin:{{ ansible_user_dir }}/.crc/bin/oc:
 
 # first one will copy ci-framework onto the ansible controller.
 # Second one will copy install_yamls onto the ansible controller.
-cifmw_reproducer_repositories:
-  - src: "{{ playbook_dir }}"
-    dest: "src/github.com/openstack-k8s-operators/"
-  - src: "{{ cifmw_install_yamls_repo }}"
-    dest: "src/github.com/openstack-k8s-operators/"
+# cifmw_reproducer_repositories:
```

Comment:
Leftovers? Care to remove or restore?

Reply:
for follow-up - this patch is needed asap for the brno meetup. It may also serve as example for ppl not reading the doc

Reply:
Keeping old code in files create confusion. I don't know if that's still applicable or not.

Reply:
removed.